packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -2197,6 +2198,36 @@ QUnit.module('aria accessibility', () => {
         assert.strictEqual($input.attr('role'), 'combobox', 'role attribute should remain assigned to the combobox after deleting');
     });
 
+    QUnit.test('component with fieldTemplate should have proper aria-labelledby attribute after interaction (T1230635)', function(assert) {
```

Comment:
We should consider the following scenarios:
- fieldTemplate is used without inputAttr option
- fieldTemplate is used with inputAttr option